python/lsst/meas/astrom/sip/leastSqFitter1d.cc

```diff
@@ -0,0 +1,76 @@
+/*
+ * LSST Data Management System
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ * See the COPYRIGHT file
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+
+#include "lsst/meas/astrom/sip/LeastSqFitter1d.h"
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace lsst {
+namespace meas {
+namespace astrom {
+namespace sip {
+
+namespace  {
+
+template <typename FittingFunc>
+void declareLeastSqFitter1d(py::module & mod, char const * name) {
```

Comment:
In most cases the string will be appended to another, in which case `std::string` is helpful, but not in this case. Nonetheless I will change it for consistency. (I wish `pybind11::class_(...)` accepted `std::string` for the name).